### DISCUSSION TITLE
Check the details of the problematic repository information

### DISCUSSION THREAD

**Discussion by mvn-quannguyen3-dn-admin** [community/community#79210] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need assistance in accessing support to retrieve detailed information about the following issue:

My company possesses a private repository, and a situation arose where an individual cloned that repository to their account and subsequently made it public on their account. 
Upon querying, I can only identify the cloning unit but cannot view the name or information of the person who performed the cloning. 
I would like your help in retrieving the details of the individual who cloned this repository publicly so that my company can localize and address the issue effectively.

Thank you for your prompt attention and support!

**Comment by mvn-quannguyen3-dn-admin** [comment]
This is an extremely serious situation in the security process that we are currently dealing with. 
I hope you will carefully review and assist me in this case.
If you need any additional information to facilitate the support process, please feel free to provide feedback.
Thank you.

**Comment by samus-aran** [comment]
Hi @mvn-quannguyen3-dn-admin 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
